### ISSUE TITLE
Timeline "Connected" status step content can be confusing for a screen reader user

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4621]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 1.3.1 Info and Relationships

**Test Environment:**
- Test URL: https://clarity.design/documentation/timeline
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. Open the test URL.
2. Select any one of the tabs provided for horizontal, vertical and angular timelines.
3. Move focus on the error state for connected step and observe the screen reader's spoken feedback.

**Expected result:**
Error message should be clear to all users.

**Actual result:**
When screen reader reads the content it is read in the following order as listed:
1. Error
2. Connected.
3. No it's not connected.

Here, each message contradicts the preceding message.

**Proposed solution:**
Change the verbiage "Connected" to "Connection status" or something similar to that which will avoid any confusion while reading through the content.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
